localization/replace/english/culture/traditions/wc_cultural_traditions_l_english.yml

```diff
@@ -28,10 +28,10 @@
  tradition_horse_lords_desc:0 "Whether upon the saddle or by one's own hooves, warriors of this culture specialise in striking foes with lightning speed and exploiting vulnerabilities in enemy formations to devastating effect."
  
  tradition_fp1_coastal_warriors_name:0 "Trials of Valour"
- tradition_fp1_coastal_warriors_desc:1 "This culture holds no honor greater than dying in glorious battle. Berserkers harness this scorn for mortality and turn themselves into unstoppable juggernauts of death through practice and ritual."
+ tradition_fp1_coastal_warriors_desc:0 "This culture holds no honor greater than dying in glorious battle. Berserkers harness this scorn for mortality and turn themselves into unstoppable juggernauts of death through practice and ritual."
```

Comment:
Do you know what the number means? It's a version of the localization.
So 0 is the first version, 1 is the second, etc. It's made for tracking changes.
Just telling. Resolve as you read. Change the version index where it's needed. It's not a critical thing, but helpful.

Reply:
Yep, I checked the other localisation files and this line (all the newly added english locs from this PR) hasn't been localised yet. This change corrects an earlier typo.